app/code/community/Nexcessnet/Alarmbell/Helper/Data.php

```diff
@@ -18,74 +18,87 @@
  * with this program; if not, write to the Free Software Foundation, Inc.,
  * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
  */
-
-class Nexcessnet_Alarmbell_Helper_Data extends Mage_Core_Helper_Abstract {
-
-	/**
-     * Formats and logs a message through Magento's logging facility
+class Nexcessnet_Alarmbell_Helper_Data extends Mage_Core_Helper_Abstract
+{
+    /**
+     * Formats and logs a message through Magento's logging facility.
+     *
+     * @param string $message
      *
-     * @param  string $message
+     * @return string
      */
-    function log($message) {
-    	// get the client IP
-		$remoteIp = Mage::helper('core/http')->getRemoteAddr();
+    public function log($message)
+    {
+        // get the client IP
+        $remoteIp = Mage::helper('core/http')->getRemoteAddr();
 
-		// get user's admin username (if logged-in)
-		$adminUsername = '';
-		$admin = Mage::getSingleton('admin/session')->getUser();
-        if (is_object($admin)) { 
-            if ($admin->getId()) { $adminUsername = $admin->getUsername(); }
+        // get user's admin username (if logged-in)
+        $adminUsername = '';
+        $admin = Mage::getSingleton('admin/session')->getUser();
+        if (is_object($admin)) {
+            if ($admin->getId()) {
+                $adminUsername = $admin->getUsername();
+            }
+        } else {
+            $adminusername = '';
         }
-        else { $adminusername = ''; }
 
         // construct log message, and log it
-        $logMessage = 'ALARMBELL ('. $remoteIp . ')';
-        if (!empty($adminUsername)) { $logMessage .= " [$adminUsername]"; }
-        $logMessage .= ': ' . $message;
+        $logMessage = 'ALARMBELL ('.$remoteIp.')';
+        if (!empty($adminUsername)) {
+            $logMessage .= " [$adminUsername]";
+        }
+        $logMessage .= ': '.$message;
```

Comment:
Good point. Maybe a bug of PSR2 fixer? Happy to roll this one back.
On Apr 23, 2016 12:28 AM, "Robbie Averill" <notifications@github.com> wrote:

> In app/code/community/Nexcessnet/Alarmbell/Helper/Data.php
> <https://github.com/nexcess/magento-alarmbell/pull/3#discussion_r60823021>
> :
>
> >
> >          // construct log message, and log it
> > -        $logMessage = 'ALARMBELL ('. $remoteIp . ')';
> > -        if (!empty($adminUsername)) { $logMessage .= " [$adminUsername]"; }
> > -        $logMessage .= ': ' . $message;
> > +        $logMessage = 'ALARMBELL ('.$remoteIp.')';
> > +        if (!empty($adminUsername)) {
> > +            $logMessage .= " [$adminUsername]";
> > +        }
> > +        $logMessage .= ': '.$message;
>
> This change actually moves further from the Zend guidelines, should stay
> as ': ' . $message
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/nexcess/magento-alarmbell/pull/3/files/1ab0c6ed80a53a9c37064717c7867597b6c3868f#r60823021>
>